server/graphql/v2/input/RecurringExpenseInput.ts

```diff
@@ -0,0 +1,20 @@
+import { GraphQLInputObjectType, GraphQLNonNull } from 'graphql';
+
+import { DateString } from '../../v1/types';
+import { RecurringExpenseInterval } from '../enum/RecurringExpenseInterval';
+
+const RecurringExpenseInput = new GraphQLInputObjectType({
+  name: 'RecurringExpenseInput',
+  fields: () => ({
+    interval: {
+      type: new GraphQLNonNull(RecurringExpenseInterval),
+      description: 'The interval in which this recurring expense is created',
+    },
+    endAt: {
```

Comment:
For consistency with what we do in events

```suggestion
    endsAt: {
```